technologies/app/gitlab/gitlab-14.7/entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -xeuo pipefail
+echo "_____________________________________________________________________________________________________"
+echo "Gitlab is initializing, it can take a few minute for the service to be up and running, please wait..."
+echo "_____________________________________________________________________________________________________"
+
```

Comment:
Shouldn't we check that environment variables are set before launching gitlab?
eg: https://github.com/saagie/technologies-community/blob/3af961685e654b3899a1535ab47cf223592bac1b/technologies/app/rshiny/rshiny-1.7.1/ressources/entrypoint.sh